### ISSUE TITLE
Check matching include and exclude parameters

### ISSUE THREAD

**Issue by sblondon** [leapfrogonline/rstr#4]
Hello,

it's possible to provide the same characters to `include` and `exclude` parameters. In such case, the characters are included. For example:
```
>>> rstr.rstr("a", include='b', exclude='b')
'aab'
>>> rstr.rstr("a", include='bc', exclude='bd')
'aabc'
```

I think about raising an exception (`SameCharacterError`?) if the intersection of `include` and `exclude` parameters is not empty. What do you think about it?

I can do the PR if you are interested.

**Comment by bjmc** [comment]
Sounds like a good idea to me. I'd be happy to review a PR for this issue.

**Comment by bjmc** [comment]
Closed by #7
